examples/nextjs-spa/README.md

```diff
@@ -0,0 +1,5 @@
+```bash
+npm run dev
+```
+
+[http://localhost:3000](http://localhost:3000)
```

Comment:
We also want to use the Ory tunnel here instead of the proxy built into the nextjs app.

Here is an example https://github.com/ory/elements/blob/main/examples/react-spa/README.md

Reply:
Specify here that `localhost:4000` is the default value for `ORY_SDK_URL`.